### PR TITLE
Adding the new monorepo UI dashboard

### PR DESCRIPTION
**Pull request by hotlib**
Merge AFTER 1.4. release

Dashboard is configurable. Version of monorepo is 'main' as there is no stable release yet (this will change in further updates).

### CODE REVIEW

File: composefiles/swarm-uniflow.yml [hunk 1]
```diff
@@ -90,7 +90,10 @@ services:
       replicas: 1
 
   dashboard:
-    image: frinx/frinx-dashboard:1.0.1
```
Comment (jrozsa) [inline composefiles/swarm-uniflow.yml:93]: Add health check pls

**Review by jrozsa — APPROVED** [review]
Looks fine to me, just one comment.

File: composefiles/swarm-uniflow.yml (hunk 1 above)
Comment (hotlib) [reply composefiles/swarm-uniflow.yml:93]: @jrozsa - what was the healthcheck until now or how should a health-check ideally look like?

Comment (jrozsa) [reply composefiles/swarm-uniflow.yml:93]: I merged the one meant for old dashboard yesterday, it should be in master.
 Just a get request and check response code is ok.

Comment (hotlib) [reply composefiles/swarm-uniflow.yml]: @jrozsa ok - I added the healthcheck and rebased on latest master

**Review by MartinSunal — CHANGES_REQUESTED** [review]
do not merge until FMv1.4 is not out and monorepo ui is not finished

**Comment by hotlib** [comment]
This was branched a and a newer PR should be done from this branch

**Pull request closed without merging**